### PR TITLE
Users/v pnizetic/existing conversions enhancements

### PR DESCRIPTION
**Pull request by nizetic**

### CODE REVIEW

File: src/common/store/user.store.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
-import { create } from 'zustand';
-
 import { getEnvs } from 'common/functions';
+import { create } from 'zustand';
 
 export const useUserStore = create(set => ({
   subscriptionKey: '',
   setSubscriptionKey: key => set(() => ({ subscriptionKey: key })),
-  geography: Object.keys(getEnvs())[0],
+  geography: localStorage.getItem('geography') || Object.keys(getEnvs())[0],
```
Comment (shnooshnoo) [inline src/common/store/user.store.js:7]: In the past we deliberately chose not to store any user data for compliance reasons. I guess it's not a big deal in this case (tho still may be worth double-checking), just wanted you to be aware of that

File: src/pages/create-manifest/create-manifest.style.js [hunk 2]
```diff
@@ -74,6 +72,18 @@ export const fileInputStyle = css`
...
 `;
 
+export const actionPanelStyle = css`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  width: 100%;
+  background-color: #ffffff;
+  box-shadow: rgba(0, 0, 0, 0.14) 0px 2px 4px;
+  padding: 12px;
```
Comment (shnooshnoo) [inline src/pages/create-manifest/create-manifest.style.js:82]: generally almost everywhere we have rems here. Which I'm not a big fan of tbh, but just kept doing it cause it was started before I joined. Won't change much most likely, but make sure this discrepancy doesn't cause any distortions, especially with different zoom values

File: src/pages/create-manifest/create-manifest.js [hunk 3]
```diff
@@ -126,28 +135,76 @@ const CreateManifestPage = () => {
...
-        text={t('process')}
-        styles={primaryButtonDisabledStyles}
-      />
-      <Link
-        to={subKey ? PATHS.CONVERSIONS : ''}
-        className={cx({ [conversionsLink]: subKey, [disabledConversionsLinks]: !subKey })}
-      >
-        {t('view.existing.conversions')}
-      </Link>
+      {false && (
```
[the remark this answers is not in the document]
Comment (shnooshnoo) [reply src/pages/create-manifest/create-manifest.js:138]: ah i see now nevermind

File: src/common/store/user.store.js (hunk 1 above)
Comment (nizetic) [reply src/common/store/user.store.js:7]: Thanks for pointing it out. I will check with the team

File: src/pages/create-manifest/create-manifest.style.js (hunk 2 above)
Comment (nizetic) [reply src/pages/create-manifest/create-manifest.style.js:82]: True. I will convert it to rem

Comment (nizetic) [reply src/pages/create-manifest/create-manifest.style.js:82]: Fixed in https://github.com/Azure/azure-maps-creator-onboarding-tool/pull/104

**Pull request merged**